### PR TITLE
Home/footer

### PR DESCRIPTION
**Pull request by fatimzzahra**
add text

### CODE REVIEW

File: Home/home.html [hunk 1]
```diff
@@ -4,6 +4,219 @@
...
+              </ul>
+            </div>
+            <div class="newsletter col-lg-4">
+              <h5 class="text-uppercase">Title</h5>
+              <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. At itaque temporibus.</p>
+              <form action="#" id="newsletter-form">
+                <div class="form-group">
+                  <input type="email" name="subscribermail" placeholder="Your Email Address">
+                  <button type="submit"> <i class="fa fa-paper-plane"></i></button>
+                  <p>blabla</p>
```
Comment (ridabensalem) [inline Home/home.html]: Hello team , well done on working on creating the nav and footer section , however I can see that "blabla" text has no meaning it is better to always use significant terms , but overall good work 👍

File: Home/home.html [hunk 2]
```diff
@@ -4,6 +4,219 @@
...
+  width: 35px;
+}
+
+
+</style>
+</head>
+<body>
+    <!-- Footer-->
+    <footer class="main-footer">
+      <!-- Main Block -->
```
Comment (ridabensalem) [inline Home/home.html:174]: Good that you used comments

File: Home/home.html [hunk 3]
```diff
@@ -4,6 +4,219 @@
...
+      <div class="main-block">
+        <div class="container">
+          <div class="row">
+            <div class="info col-lg-4">
+              <div class="logo"><h2>LAEDEX</h2></div>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing.</p>
```
Comment (ridabensalem) [inline Home/home.html]: Here use a good text

**Review by ridabensalem — CHANGES_REQUESTED** [review]
Hi @fatimzzahra @AmineBaray @MustaphaJAM Hi Team, 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights
- Very beautiful UI
- no linters error
- tests working properly

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please make sure to fix the MIT license link at the bottom of your Readme file
- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Home/home.html [hunk 4]
```diff
@@ -4,9 +4,168 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
-
```
Comment (MustaphaJAM) [inline Home/home.html]: hi team 👋,I see that we are almost done with our small project but I see her a small problem, this line is repeated, the overall is great.👌

File: Home/home.html [hunk 5]
```diff
@@ -89,10 +248,55 @@ <h5 class="card-title">Fatim ezzahra Abi</h5>
                   </div>
               </div>
           </div>
-    </main>  
-
-
+    </main>
```
Comment (MustaphaJAM) [inline Home/home.html]: I see that this element is repeated to </main> 👍.

File: Home/home.html [hunk 6]
```diff
@@ -89,10 +248,55 @@ <h5 class="card-title">Fatim ezzahra Abi</h5>
...
+                  <p>you can contact us</p>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </footer>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
-
```
Comment (MustaphaJAM) [inline Home/home.html:96]: this line is empty

**Review by MustaphaJAM — CHANGES_REQUESTED** [review]
Hi  @AmineBaray  @fatimzzahra  , 

Good job so far. !
There are some issues that you still need to work on to go to the next task  but you are almost there!
mario_ladders



## Highlights 
- Clean testing code 

## Required Changes ♻
* You really did a great job but there are some features you need to improve

- [ ] Please make sure to remove those repetitions and empty spaces 
- [ ] Please check it

**Pull request closed without merging**